front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -54,11 +67,24 @@ export default function Project() {
   const [studiesList, setStudiesList] = useState<StudyResult[]>([]);
   const [filter, setFilter] = useState('');
   const [filterChips, setFilterChips] = useState('');
-  const [sortOption, setSortOption] = useState('LastModifiedDesc');
+  const [sortOption, setSortOption] = useState<SortOptions>('LastModifiedDesc');
   const [imageUrl, setImageUrl] = useState('');
   const dispatch = useDispatch();
   const navigate = useNavigate();
-  const projectID = useSelector(getProjectID);
+  const projectId = useSelector(getProjectID);
+  const {
+    data: currentProject,
+    isLoading,
+    isError,
+    error: projectError,
+  } = osrdEditoastApi.useGetProjectsByProjectIdQuery({ projectId: projectId as number });
+  const [postSearch] = osrdEditoastApi.usePostSearchMutation();
+  const [getStudies] = osrdEditoastApi.useLazyGetProjectsByProjectIdStudiesQuery();
+
+  if (isError) {
+    console.error('getProject Error : ', projectError);
+    return <span className="mt-5">An error occured</span>;
+  }
```

Comment:
- same as above for console.error
- NEVER put text in code, always use translations

Here, I prefer fire a notification toast rather than replace whole page by return.

Reply:
Done